docs/build/evm.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Flow EVM
+sidebar_label: Flow EVM
+sidebar_position: 10
+---
+
+# About EVM on Flow
+
+Flow is a secure Layer 1 blockchain committed to driving Web3 adoption into the mainstream. The team behind the blockchain not only coined the term NFT but made strides in opening up web3 to the world through Disney Pinnacle, NBA Topshot and Doodles. A significant milestone in this journey is the introduction of Ethereum Virtual Machine (EVM) equivalency i 2024. 
```

Comment:
```suggestion
Flow is a secure Layer 1 blockchain committed to driving Web3 adoption into the mainstream. The team behind the blockchain not only coined the term NFT but made strides in opening up web3 to the world through Disney Pinnacle, NBA Topshot and Doodles. A significant milestone in this journey is the introduction of Ethereum Virtual Machine (EVM) equivalency in 2024. 
```

Reply:
Is it really necessary to rehash the history here?